CONTRIBUTING.md

```diff
@@ -53,6 +61,27 @@ rebuild and try again.
 npm rebuild
 ```
 
+#### Install composer
+
+We use composer for some critical packages. Please [install composer](https://getcomposer.org/download/) (if you do not already have it).
+
+#### Install composer packages
+
+With composer in place, let's get those composer packages:
+```
+composer install
+```
+
+#### Install npm and composer packages in common
+
+One of our submodules is Tribe Common, which is a shared set of libraries in use by our plugins (see also [Event Tickets](https://github.com/moderntribe/event-tickets)). It uses npm and composer as well, so, let's get those as well.
+
+```
+cd common
+npm install
+composer install
```

Comment:
`common` has sub-modules too:
```suggestion
composer install
git submodule update --init --recursive
```

Reply:
the --recursive should cover that though, right?

Reply:
Yup, it will. We don't need the `git submodule update --init --recursive` here.